packages/commons-test/src/containerTestUtils.ts

```diff
@@ -119,3 +121,19 @@ export const mailpitContainer = (): GenericContainer =>
 
 export const redisContainer = (): GenericContainer =>
   new GenericContainer(TEST_REDIS_IMAGE).withExposedPorts(TEST_REDIS_PORT);
+
+/**
+ * Starts a container that exposes an AWS SES v2 compatible API.
+ *
+ * This container is used to test the email sending functionality of the
+ * AWS SES services.
+ *
+ * @returns A promise that resolves to the started test container.
+ */
+export const awsSESContainer = (): GenericContainer =>
+  new GenericContainer("node:20")
```

Comment:
Could you create a const with the image as the other containers?

```suggestion
  new GenericContainer(TEST_AWS_SES_IMAGE)
```